Mathmagician/Commands/Integer.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mathmagician.Commands
+{
+    public class Integer : Command
+    {
+        public override void CreateNumbers(int quantityOfNumsToGenerate)
+        {
+            //Create integers to add to Numbers List
+            for (int i = 0; i != quantityOfNumsToGenerate; i++)
+            {
+                Numbers.Add(i);

```

Comment:
I should have been more clear, I meant generating negative integers, but not using them as 'i'. The idea I had in my head is probably too much work for the exercise, but would involve using the for loop to create the initial number and then randomly make them a positive or negative number before adding to the list.